app/src/main/admin/actions/usersActionCreators.ts

```diff
@@ -24,7 +25,8 @@ export const usersActionCreators = {
         }
     },
 
-    createUser(name: string, email: string, username: string) {
+    createUser(values: CreateUserFormFields) {
+        const {name, email, username} = values;
```

Comment:
I changed the signature of this method because named parameters are less error prone than positional.